### PR TITLE
chore: upgrade vision samples to new format

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: vision/README.md [hunk 1]
```diff
@@ -28,52 +28,15 @@ This simple command-line application demonstrates how to invoke
...
-  product-set-add-product               Add product to a product set
-  product-set-list-products             List products in a product set
-  product-set-remove-product            Remove product from a product set
-  product-purge-orphan                  Delete all products not in any product sets
-  product-purge-products-in-product-set Delete all products not in any product set
-```
-
-7. Run `php vision.php COMMAND --help` or `php product_search.php COMMAND --help` to print information about the usage of each command.
-
+5.  Execute the snippets in the [src/](src/) directory by running
```
Comment (vishwarajanand) [inline vision/README.md:31]:
```suggestion
6.  Execute the snippets in the [src/](src/) directory by running
```

File: vision/README.md [hunk 2]
```diff
@@ -28,52 +28,15 @@ This simple command-line application demonstrates how to invoke
...
+    `php src/SNIPPET_NAME.php`. The usage will print for each if no arguments
+    are provided:
+    ```sh
+    $ php src/detext_face.php
+    Usage: php src/detext_face.php PATH
```
Comment (vishwarajanand) [inline vision/README.md:36]:
```suggestion
    Usage: php src/detext_face.php <PATH>
```

File: vision/test/visionTest.php [hunk 3]
```diff
@@ -169,14 +165,14 @@ public function testLogoCommandGcs()
         $bucketName = $this->requireEnv('GOOGLE_STORAGE_BUCKET');
 
         $path = 'gs://' . $bucketName . '/vision/logo.jpg';
-        $output = $this->runCommand('logo', ['path' => $path]);
+        $output = $this->runFunctionSnippet('detect_logo_gcs', ['path' => $path]);
         $this->assertStringContainsString('Google', $output);
     }
 
     public function testLocalizeObjectCommand()
```
Comment (vishwarajanand) [inline vision/test/visionTest.php:172]: You might want to remove the term `localize` from test method name? 
Something like:
```suggestion
    public function testDetectObjectCommand()
```

**Review by vishwarajanand — APPROVED** [review]

File: vision/test/visionTest.php (hunk 3 above)
Comment (bshaffer) [reply vision/test/visionTest.php:172]: You're right - I think the sample was renamed (if you look at the region tag in `detect_object.php`, it's still using `localized`). 

I agree that "detect" is preferred. I'll update line 183 as well!

**Review by dwsupplee — APPROVED** [review]
Looks nice!

**Pull request merged**